docs/samples/browser-plugin-meetings/index.html

```diff
@@ -210,6 +210,7 @@ <h2 class="collapsible">Meeting Management</h2>
                         <option value="personID">Person ID</option>
                         <option value="roomID">Room ID</option>
                         <option value="sipURI">SIP URI</option>
+                        <option value="conversationURL">Conversation URL</option>
```

Comment:
lets change the value to CONVERSATION_URL

Reply:
remove all the other values except the conversationUrl

Reply:
Done, changed the value to CONVERSATION_URL and removed other values.